client/components/App.js

```diff
@@ -1,51 +1,67 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+import Cookies from 'js-cookie';
 import Menu from './Menu';
 import Login from './Login';
 import ProductSingle from './ProductSingle';
 import Nav from './Nav';
 import Signup from './Signup';
 import Home from './Home';
 import Order from './Order';
-import Checkout from './Checkout';
-import AdminProducts from './AdminProducts';
+import { setSessionThunk } from '../store';
 
-const App = () => {
-  return (
-    <Router>
-      <header id="main-header">
-        <div id="site-name" className="flex-container">
-          Lovin Today
-        </div>
-        <Route render={location => <Nav location={location} />} />
-      </header>
-      <section id="content" className="container-fluid">
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route exact path="/menu" component={Menu} />
-          <Route path="/menu/product/:productId" component={ProductSingle} />
-          <Route path="/login" component={Login} />
-          <Route path="/signup" component={Signup} />
-          <Route path="/bag" component={Order} />
-          <Route path="/checkout" component={Checkout} />
```

Comment:
Shouldn't we keep the Routes for Checkout and AdminProducts?

Reply:
Yes, something happened during merge that messed things up.  Fixing ...